test/spec/geo/ui/time_slider.spec.js

```diff
@@ -63,6 +63,20 @@ describe('cdb.geo.ui.TimeSlider', function() {
       });
     });
 
+    describe("given a step that spans more than 48 hours", function() {
+      beforeEach(function() {
+        var start = 1423699205000;
+        var end = 1424649534000;
+        view.torqueLayer.getTimeBounds = function(){return {start: start, end: end}};
+        view.torqueLayer.options = {steps: 5};
+        formatter = view.formatterForRange.bind(view)(start, end);
+      });
+      it("should return a formatter function that defines a two-date range", function() {
+        var moment = new Date(1424269402400);
+        expect(formatter.bind(view)(moment)).toEqual("Feb 18 - Feb 20");
```

Comment:
.bind is best used when you want to make a partial application of the function (predefined params), if you just want to call the function with a certain context it's better to use .call/.apply (depending on what kind of arguments you want to give it). So to make it clearer I'd recommend to just use:
``formatter.call(view, moment)`